bin/reporttests.jl

```diff
@@ -79,10 +79,6 @@ if abspath(PROGRAM_FILE) == @__FILE__()
     )
     cmd = TestReports.gen_command(runner_code, ``, coverage)
 
-    try
-        run(cmd)
-    catch e
-        e isa ProcessFailedException && exit(only(e.procs).exitcode)
-        rethrow()
-    end
+    p = run(ignorestatus(cmd))
+    success(p) || exit(p.exitcode)
```

Comment:
we can probably just do
```suggestion
    exit(p.exitcode)
```

Since if it is a success the exit code is zero.

Reply:
You are correct. Feels a little weird to me but the logic is the same.